pages/common/starship.md

```diff
@@ -0,0 +1,36 @@
+# starship
+
+> The minimal, blazing-fast, and infinitely customizable prompt for any shell.
+> More information: <https://starship.rs/>.
+
+- Print the shell function used to execute starship:
+
+`starship init {{bash|fish|zsh}`
+
+- Explain the current showing modules:
+
+`starship explain`
+
+- Print the computed starship configuration (use `--default` to print default configuration instead):
+
+`starship print-config`
+
+- List supported modules:
+
+`starship module --list`
+
+- Edit the starship configuration:
```

Comment:
Does it use the systems default or the one in `$EDITOR`?

Reply:
It uses `$EDITOR` if it's set, otherwise the system's default (`vim` for me, on macOS).

Reply:
Should we maybe update the description to `Edit the starship configuration in the default editor`?

Reply:
Sounds good to me. 👍🏻

Reply:
```suggestion
- Edit the starship configuration in the default editor:
```